static/app/views/issueDetails/traceTimeline/traceTimeline.spec.tsx

```diff
@@ -152,4 +152,71 @@ describe('TraceTimeline', () => {
     render(<TraceTimeline event={event} />, {organization});
     expect(await screen.findByLabelText('Current Event')).toBeInTheDocument();
   });
+
+  it('skips the timeline and shows related issues (2 issues)', async () => {
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/events/`,
+      body: issuePlatformBody,
+      match: [MockApiClient.matchQuery({dataset: 'issuePlatform'})],
+    });
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/events/`,
+      body: discoverBody,
+      match: [MockApiClient.matchQuery({dataset: 'discover'})],
+    });
+    // I believe the call to projects is to determine what projects a user belongs to
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/projects/`,
+      body: [],
+    });
+
+    // Enable related issues feature flag
+    organization.features = ['related-issues-issue-details-page'];
```

Comment:
We want to avoid mutating in tests, since it will make the order of tests matter. Better to add the flag like this:

```ts
render(..., {organization: OrganizationFixture({features: [...]})})
```